.github/workflows/merge-conflict-auto-label.yml

```
@@ -3,6 +3,8 @@ on:
   push:
     branches:
       - main
+  schedule:
+    - cron: "20 4 * * *"
```

Setting this to run pretty soon for testing ^^

```suggestion
    - cron: "40 20 * * *"
```